### PR TITLE
[Tests-Only]enable drone

### PR DESCRIPTION
**Pull request by jasson99**
## Description:
 This PR enables drone CI to run unit tests in CI.

### CODE REVIEW

File: .drone.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+name: lint
+
+trigger:
+  event:
+    include:
+      - pull_request
+      - tag
+
+steps:
+  - name: build
```
Comment (dpakach) [inline .drone.yml:13]: change this name.

File: .drone.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+trigger:
+  event:
+    include:
+      - pull_request
+      - tag
+
+steps:
+  - name: unit test
+    image: node:12.14.1
```
Comment (dpakach) [inline .drone.yml:33]: Lets use more recent version of node here. current lts is 14

Comment (jasson99) [reply .drone.yml]: fixed

File: .drone.yml (hunk 1 above)
Comment (jasson99) [reply .drone.yml]: fixed

**Review by individual-it — APPROVED** [review]

**Pull request merged**